cddl/corim-map.cddl

```diff
@@ -3,7 +3,6 @@ corim-map = {
   &(tags: 1) => [ + $concise-tag-type-choice ]
   ? &(dependent-rims: 2) => [ + corim-locator-map ]
   ? &(profile: 3) => $profile-type-choice
-  ? &(rim-validity: 4) => validity-map
-  ? &(entities: 5) => [ + corim-entity-map ]
+  ? &(entities: 4) => [ + corim-entity-map ]
```

Comment:
Please don't renumber existing map keys as that makes existing files invalid.

Reply:
Done.